src/base/bittorrent/session.cpp

```diff
@@ -4320,7 +4320,7 @@ void Session::handleAlert(const lt::alert *a)
     {
         switch (a->type())
         {
-#if (LIBTORRENT_VERSION_NUM >= 20003)
+#if (LIBTORRENT_VERSION_NUM >= 20004)
```

Comment:
"Find and Replace" isn't generally enough.

Reply:
@glassez 

> "Find and Replace" isn't generally enough.

You mean you want this particular one to stay at `>= 20003`? Then this would become an outlier in comparison to the others that are `>= 20000`, This is a discrepancy that we can avoid by keeping everything in lockstep with the minimum required version in the build script (in this case, `>= 20004`). And it better conveys the fact that we're really serious about the minimum required version.

> Why change them every time we we raise the minimum version within a branch?

Why not? As mentioned above, we can keep everything more uniform. And it's quite easy: every time we bump the version, we can just find and replace `LIBTORRENT_VERSION_NUM >= 2.*)` with the new number. Once we finally drop support for 1.2.x, we'll stop doing this.

Reply:
>You mean you want this particular one to stay at `>= 20003`? 

Not only this one but all the similar (`LIBTORRENT_VERSION_NUM >= 2.*`).

>Then this would become an outlier in comparison to the others that are `>= 20000`, This is a discrepancy that we can avoid by keeping everything in lockstep with the minimum required version in the build script (in this case, `>= 20004`). 

This is still not about our support for this or that version of libtorrent, but about the presence of some functions in this or that version of libtorrent. Let's leave it as it is. I insist.

Reply:
@glassez fixed. there are no other instances that I found to delete/modify that meet your criteria.